devops/openshift/forms-deploy.yml

```diff
@@ -216,9 +233,6 @@ parameters:
     description: FORMIO_ROOT_PASSWORD
     generate: expression
     from: "[a-zA-Z0-9]{16}"
```

Comment:
If this is not removed, will it supersede the assigned valued from agrument?

Reply:
Nice catch. I removed the auto-generated password.